### ISSUE TITLE
breqn not found

### ISSUE THREAD

**Issue by jgoddard9** [wspr/breqn#14]
The LaTeX package breqn is needed for a cadabra installation. I have downloaded and unzipped the breqn folder, compiled the .dtx file with LaTeX, moved the folder to ~/Library/texmf, and run texhash in Terminal on Mac OS 10.11.5. When I try to run ./configure on cadabra, I am getting the error message:

'configure: error: Need the breqn LaTeX package; get it from ftp://ftp.ams.org/pub/tex/ .'

I note that /opt/local/share/texmf-texlive/tex/latex/breqn exists and contains all the style files.

I have reinstalled MacTex in case that might have been the problem, but that did not help. Any idea what the problem might be? I am not sure whether it is a breqn or cadabra issue. 

Thanks, Joe Goddard

**Comment by wspr** [comment]
Hi Joe,

You write:

> I note that /opt/local/share/texmf-texlive/tex/latex/breqn exists and contains all the style files. I have reinstalled MacTex in case that might have been the problem, but that did not help. Any idea what the problem might be? I am not sure whether it is a breqn or cadabra issue. 


Unfortunately I don’t have resources to debug specific installations, but it looks like you’ve got multiple versions of TeX installed. The path `/opt/local/…` is where MacPorts (I think) installs TeX, and I suspect this is messing things up. I’d recommend deleting the MacPorts distribution and sticking with MacTeX in all cases.

Hope this helps.

**Comment by jgoddard9** [comment]
Hi Will,  I recently uninstalled MacPorts but reinstalled because there 
were some broken port files.
This did not solve the problem. Do you know whether removing MacPorts 
completely will remove all the TeX
iit installed? Pardon my ignorance and thanks for the prompt reply.  Joe

**Comment by jgoddard9** [comment]
Good day, Will, I uninstalled MacPorts and tried installing cadabra from 
the install package with ./configure, etc., but
I encountered a host of dependencies that required further packages with 
dependencies, ad infinitum. Hence,
I reinstalled MacPorts and tried installing cadabra with port. However, 
port installed a bunch of texlive packages
that probably conflict with my MacTex installation, and the install of 
cadabra failed. I have pretty much given
up. Joe
